### PR TITLE
add special cases for the pre-defined atoms 0 and 1

### PR DESCRIPTION
**Pull request by arvidn**

### CODE REVIEW

File: src/serialize.rs [hunk 1]
```diff
@@ -134,6 +134,12 @@ pub fn node_from_stream<T: Allocator>(allocator: &T, f: &mut dyn Read) -> std::i
                     ops.push(ParseOp::Cons);
                     ops.push(ParseOp::SExp);
                     ops.push(ParseOp::SExp);
+                } else if b[0] == 0x00 {
+                    values.push(allocator.null());
+                } else if b[0] == 0x01 {
+                    values.push(allocator.one());
+                } else if b[0] == 0x80 {
```
Comment (richardkiss) [inline src/serialize.rs:141]: This is a lone zero, which is NOT a null

Comment (richardkiss) [reply src/serialize.rs:141]: Sorry, `0x80` is an empty string... it's `0x00` that a lone zero (which is not a null). I think.

Comment (arvidn) [reply src/serialize.rs:139]: fixed

File: src/more_ops.rs [hunk 2]
```diff
@@ -403,8 +403,8 @@ pub fn op_divmod<T: Allocator>(args: &Node<T>) -> Response<T::Ptr> {
         let q = &a0 / &a1;
         let r = &a0 - &a1 * &q;
 
-        let signed_quotient = (a0.sign() == Sign::Minus || a1.sign() == Sign::Minus)
-            && a0.sign() != a1.sign();
+        let signed_quotient =
+            (a0.sign() == Sign::Minus || a1.sign() == Sign::Minus) && a0.sign() != a1.sign();
```
Comment (arvidn) [inline src/more_ops.rs:407]: this formatting issue slip through earlier

**Review by richardkiss — APPROVED** [review]

**Pull request merged**